inject-generator/src/main/java/io/avaje/inject/generator/Util.java

```diff
@@ -284,7 +284,7 @@ static String commonParent(String currentTop, String aPackage) {
   static String getNamed(Element p) {
     final NamedPrism named = NamedPrism.getInstanceOn(p);
     if (named != null) {
-      return named.value();
+      return named.value().replace("\"", "\\\"");
```

Comment:
Why is this replace of quotes here?

Reply:
The reason is to bring it in line with the annotation qualifiers